python-lib/lal/classifiers/text_classifier.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+
+import pandas as pd
+
+TEXT_COLUMN_DEFAULT_LABEL = 'text'
+
+
+from lal.classifiers.base_classifier import TableBasedDataClassifier
+
+
+class TextClassifier(TableBasedDataClassifier):
+    logger = logging.getLogger(__name__)
+
+    def __init__(self, initial_df, queries_df, config=None):
+        self.__initial_df = initial_df
+        self.text_column = config["text_column"]
+        super(TextClassifier, self).__init__(queries_df, config)
+
+    def get_initial_df(self):
+        return self.__initial_df
+
+    def clean_data_to_save(self, lab):
+        return {
+            'text': lab['text'],
+            'startId': lab['startId'],
+            'endId': lab['endId'],
+            'label': lab['label']
+        }
+
+    def get_relevant_config(self):
+        return {"text_column": self.text_column}
```

Comment:
do we need the new method? there's already a `config` property in the parent class. So you could access this fielf like `classifier.config.get('text_column')`

Reply:
The goal of this method is to be generic and present in each classifier. As the text labeling sends specific config to the front that do not make sense for other labeling components, I thought it would be useful to have a method that returns specific config for a labeling type.
Without it, I would have to use a 
`if (type == "text"):
       this.config['text_column'] = classifier.config.get('text_column')
` but it would not be very generic if other components need specific parameters as well.
Does it make sense ?

Reply:
hmm, not sure I get it, where would you have to call this code?
`if (type == "text"): this.config['text_column'] = classifier.config.get('text_column') `

from what I see you're using `get_relevant_config` to populate `classifierConfig` that's later used in:
```
                    <TextArea v-if="type === 'text'"
                                 :text="item.data.raw[config.classifierConfig.text_column]"
```

but you could just refer to that field as:
`config.text_column`


Let me know if you'd like to discuss that via zoom

Reply:
Changed in 8168afd.

I add to do a little bit different from what we discussed of, as I use the parameters given in the webapp.json  into my backend. Thus, I have to get the project variable if there is no local variable in the backend. So I keep sending the same data to front-end, however, I flat the config varaible in front so that the information appear only once.